azure-pipelines/build.yaml

```diff
@@ -74,7 +74,7 @@ steps:
   # hence don't set PYTHONPATH
   - bash: |
       source activate InnerEye
-      pytest ./Tests/ -m "not gpu and not azureml and not after_training_single_run and not after_training_ensemble_run and not inference" --doctest-modules --junitxml=junit/test-results.xml --cov=. --cov-report=xml --cov-report=html -n 2 --dist=loadscope --verbose
+      pytest ./Tests/ -m "not gpu and not azureml and not after_training_single_run and not after_training_ensemble_run and not inference and not after_training_2node" --doctest-modules --junitxml=junit/test-results.xml --cov=. --cov-report=xml --cov-report=html -n 2 --dist=loadscope --verbose
```

Comment:
The -m option in pytest allows parentheses, so this is valid: -m "not (x or y or z)"